python/setup.py

```diff
@@ -39,24 +39,4 @@
     # include neither binaries
     pass
 
-setup(
```

Comment:
note that you have more things to setup in the current version of the pythonize branch